content/cloud-docs/api-docs/run.mdx

```diff
@@ -67,14 +67,14 @@ The run state is found in `data.attributes.status`, and you can reference the fo
 | `applied`              | The applying phase of a run has completed.                                                                                                                                                                                                                                                                                                                                                                                         |
 | `discarded`            | The run has been discarded. This is a final state.                                                                                                                                                                                                                                                                                                                                                                                 |
 | `errored`              | The run has errored. This is a final state.                                                                                                                                                                                                                                                                                                                                                                                        |
 | `canceled`             | The run has been canceled.                                                                                                                                                                                                                                                                                                                                                                                                         |
-| `force_canceled`       | The run has been canceled, forcefully.                                                                                                                                                                                                                                                                                                                                                                                              |
+| `force_canceled`       | The run has been canceled, forcefully.                                                                                                                                                                                                                                                                                                                                                                                             |
```

Comment:
Can we provide the tiniest bit more here. How is this different from just being canceled? Is it because of who is doing the canceling? or....what do we mean exactly? Also, should we say that this is a final state, like the others?

Reply:
I will add more. It means that the run could not be canceled gracefully and it was force-canceled by a workspace admin.